modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -34,6 +40,28 @@ spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
 spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-uyuni-client-x86_64'
```

Comment:
This should be enough.
```
spacewalk-common-channels centos7 centos7-updates centos7-uyuni-client
```

But I am not sure if this is what we do. On SUSE Manager content, should we use the RES7 client tools?